crates/vault-registry/src/lib.rs

```diff
@@ -466,8 +466,11 @@ pub mod pallet {
     pub(super) type LiquidationVaultAccountId<T: Config> = StorageValue<_, T::AccountId, ValueQuery>;
 
     #[pallet::storage]
-    #[pallet::getter(fn liquidation_vault)]
-    pub(super) type LiquidationVault<T: Config> = StorageValue<_, SystemVault<BalanceOf<T>>, ValueQuery>;
+    pub(super) type LiquidationVault<T: Config> =
+        StorageMap<_, Blake2_128Concat, CurrencyId<T>, DefaultSystemVault<T>, OptionQuery>;
+
+    #[pallet::storage]
+    pub(super) type WrappedDebt<T: Config> = StorageValue<_, Wrapped<T>, ValueQuery>;
```

Comment:
My concern is that we're conflating terminology, what was once tracked under `issued_tokens` for the liquidation vault is now `WrappedDebt`. Something completely explicit would be `LiquidationVaultIssuedTokens`.

Reply:
That would be easy yo confuse with the individual liquidation vaults issued tokens. Something completely explicit would be TotalIssuedTokensAcrossAllLiquidationVaults. But that's a bit much.. We could also remove the variable altogether, it was used to optimize a singl function, but I think it's only used in a RPC so the performance doesn't matter too much

Reply:
Or `TotalLiquidationIssuedTokens` like we have similar to `TotalUserVaultCollateral` (on my phone so can't remember if that's the exact naming). Anyway we should check if this RPC is used anywhere because I'd be in favour of removing as many unnecessary calls as possible.

Reply:
Surprisingly, the `get_total_issued_tokens` was not even an RPC (anymore?), so I deleted it now, together with the wrapped debt